SimPEG/Mesh/TensorMesh.py

```diff
@@ -592,6 +592,30 @@ def edge(self):
         return self._edge
 
     @property
+    def edgeBoundaryInd(self):
```

Comment:
No, it has not. This is for the `InteriorBoundary` formulation which I haven't implement.